packages/e2e-tests/src/specs/editor/editor.js

```diff
@@ -42,4 +42,13 @@ describe('Story Editor', () => {
     await percySnapshot(page, 'Empty Editor on RTL');
     await deactivateRTL();
   });
+
+  it('should cross origin isolation be enabled in the browser', async () => {
+    await createNewStory();
+
+    const crossOriginIsolated = await page.evaluate(
+      () => window.crossOriginIsolated
+    );
+    expect(crossOriginIsolated).toBeTrue();
+  });
```

Comment:
Now that I think about it, it would be very beneficial to have a separate test in addition to this one were:

1. User goes to Settings
2. User disables video optimization
3. User goes to editor
4. `window.crossOriginIsolated` is `false`

That'd really give us some assurance that it will be disabled as expected, which is what we communicate in support.